build/dockerfiles/Dockerfile

```diff
@@ -23,14 +24,16 @@ RUN TAG=${PATCHED_IMAGES_TAG} \
     REGISTRY=${PATCHED_IMAGES_REG} \
     ./update_devfile_patched_image_tags.sh
 RUN ./check_mandatory_fields.sh devfiles
+RUN if [[ ${USE_DIGESTS} == "true" ]]; then ./write_image_digests.sh devfiles; fi
```

Comment:
I don't see this `write_image_digests.sh` file in the repo

Reply:
Since we're not setting USE_DIGESTS to true, I left it out.

Reply:
Since we never set `USE_DIGESTS`, why are we adding the code here at all?

Reply:
Just copying as-is from the Che repo

Reply:
In that case I think we should include the write image digest script for completeness.  We should either have it all-in or all-out

Reply:
Fair enough, I'll add the script.